ispdb/sakunet.ne.jp.xml

```diff
@@ -4,18 +4,26 @@
     <domain>sakunet.ne.jp</domain>
     <displayName>佐久ケーブルテレビ</displayName>
     <displayShortName>Saku-Net</displayShortName>
+    <incomingServer type="imap">
+      <hostname>imap.sakunet.ne.jp</hostname>
+      <port>993</port>
+      <socketType>SSL</socketType>
+      <username>%EMAILLOCALPART%</username>
```

Comment:
thought: This provider seems to use CloudMail, so I'm wondering if we should use the whole email address too (as per https://github.com/thunderbird/autoconfig/pull/105#discussion_r1535449259).